src/deluge/model/instrument/midi_instrument.cpp

```diff
@@ -226,7 +226,8 @@ void MIDIInstrument::monophonicExpressionEvent(int32_t newValue, int32_t whichEx
 	}
 
 	case 1:
-		midiEngine.sendCC(masterChannel, 74, (newValue >> 25) + 64, channel);
+		//send CC1 for monophonic expression - monophonic synths won't do anything useful with CC74
+		midiEngine.sendCC(masterChannel, 1, (newValue >> 25) + 64, channel);
```

Comment:
this seems like it has potential to break setups, can/should we make it configurable?

Reply:
That's a fair point. I don't know of a non MPE synth that uses CC74 as a general mod destination but they might exist

Reply:
(discussed on Discord) - That seems fine, we can add a menu item for this in the future. Required for nymphes, I think.